js/classes/entities/midground/weapon/projectile.js

```diff
@@ -166,7 +168,7 @@ class Projectile {
             GAME.entities.forEach((entity) => {
```

Comment:
This is great! Could we make this its own function or put a comment saying why or what its doing?